### ISSUE TITLE
Fresh Flutter project, added tflite_flutter to dependencies, debug hangs on Running pod install... (VS Code / Terminal)

### ISSUE THREAD

**Issue by davidd647** [tensorflow/flutter-tflite#122]
No errors...

The only addition to the project was:
`tflite_flutter: ^0.10.1`
in the pubspec.yaml file.

I tried running a `flutter run` with my phone plugged in.

It seems to succesfuly Sign the iOS app using my developer identity, and then this text stands out to me:
```
Found plugin tflite_flutter at
/Users/daviddales/.pub-cache/hosted/pub.dev/tflite_flutter-0.10.1/
[   +4 ms] executing: which pod
[   +7 ms] executing: pod --version
[ +330 ms] 1.11.3
[        ] Running pod install...
```

With a big array of other stuff... I guess I'll post it here in case it's of any help
```
[ +191 ms] [
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "16.4 (20E247)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPad13,17",
                        "identifier" : "CB240382-4D02-4F10-B778-EFD1AA66E4DA",
                        "architecture" : "arm64",
                        "modelUTI" : "com.apple.ipad-air5-1",
                        "modelName" : "iPad Air (5th generation)",
                        "name" : "iPad Air (5th generation)",
                        "ignored" : false
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "16.4 (20E247)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPad14,5",
                        "identifier" : "9DFE8A7C-49E7-478B-9A6E-2CF2EDFB2ECA",
                        "architecture" : "arm64",
                        "modelUTI" : "com.apple.ipad-pro-12point9-6th-1",
                        "modelName" : "iPad Pro (12.9-inch) (6th generation)",
                        "name" : "iPad Pro (12.9-inch) (6th generation)",
                        "ignored" : false
                      },
                      {
                        "ignored" : false,
                        "modelCode" : "iPhone12,1",
                        "simulator" : false,
                        "modelName" : "iPhone 11",
                        "operatingSystemVersion" : "16.5.1 (20F75)",
                        "identifier" : "00008030-000E11EE3E3A802E",
                        "platform" : "com.apple.platform.iphoneos",
                        "architecture" : "arm64e",
                        "interface" : "usb",
                        "available" : true,
                        "name" : "David’s iPhone (2)",
                        "modelUTI" : "com.apple.iphone-11-1"
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "16.4 (20E247)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPad14,1",
                        "identifier" : "56A61AD5-89C1-4159-8E9F-FE1CCCB0A5AB",
                        "architecture" : "arm64",
                        "modelUTI" : "com.apple.ipad-mini6-1",
                        "modelName" : "iPad mini (6th generation)",
                        "name" : "iPad mini (6th generation)",
                        "ignored" : false
                      },
                      {
                        "ignored" : false,
                        "modelCode" : "Watch6,7",
                        "simulator" : false,
                        "modelName" : "Apple Watch Series 7",
                        "error" : {
                          "code" : 19,
                          "failureReason" : "",
                          "description" : "David’s Apple Watch has recently restarted.",
                          "recoverySuggestion" : "To use David’s Apple Watch with Xcode, unlock it
                          and its companion iPhone",
                          "domain" : "DVTDeviceIneligibilityErrorDomain"
                        },
                        "operatingSystemVersion" : "9.5.2 (20T571)",
                        "identifier" : "00008301-C89C7C923A98C02E",
                        "platform" : "com.apple.platform.watchos",
                        "architecture" : "armv7k",
                        "interface" : "proxied",
                        "companionDevice" : {
                          "ignored" : false,
                          "modelCode" : "iPhone12,1",
                          "simulator" : false,
                          "modelName" : "iPhone 11",
                          "operatingSystemVersion" : "16.5.1 (20F75)",
                          "identifier" : "00008030-000E11EE3E3A802E",
                          "platform" : "com.apple.platform.iphoneos",
                          "architecture" : "arm64e",
                          "interface" : "usb",
                          "available" : true,
                          "name" : "David’s iPhone (2)",
                          "modelUTI" : "com.apple.iphone-11-1"
                        },
                        "available" : false,
                        "name" : "David’s Apple Watch",
                        "modelUTI" : "com.apple.watch-series7-1"
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "16.4 (20E247)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPhone15,3",
                        "identifier" : "E964F62E-2958-4A61-B108-1F91C9DA719E",
                        "architecture" : "arm64",
                        "modelUTI" : "com.apple.iphone-14-pro-max-1",
                        "modelName" : "iPhone 14 Pro Max",
                        "name" : "iPhone 14 Pro Max",
                        "ignored" : false
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "16.4 (20E247)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPad14,3",
                        "identifier" : "4839B024-6422-49C6-8E6A-6AC18B39CFF9",
                        "architecture" : "arm64",
                        "modelUTI" : "com.apple.ipad-pro-11-4th-1",
                        "modelName" : "iPad Pro (11-inch) (4th generation)",
                        "name" : "iPad Pro (11-inch) (4th generation)",
                        "ignored" : false
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "16.4 (20E247)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPhone15,2",
                        "identifier" : "26813DDD-B318-43E2-A785-816047B0AC82",
                        "architecture" : "arm64",
                        "modelUTI" : "com.apple.iphone-14-pro-1",
                        "modelName" : "iPhone 14 Pro",
                        "name" : "iPhone 14 Pro",
                        "ignored" : false
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "16.4 (20E247)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPhone14,6",
                        "identifier" : "54023D36-9931-43A3-A2D8-4DE2005B867D",
                        "architecture" : "arm64",
                        "modelUTI" : "com.apple.iphone-se3-1",
                        "modelName" : "iPhone SE (3rd generation)",
                        "name" : "iPhone SE (3rd generation)",
                        "ignored" : false
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "16.4 (20E247)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPhone14,8",
                        "identifier" : "96C6FF5D-1CC7-4D31-B628-116008BF5261",
                        "architecture" : "arm64",
                        "modelUTI" : "com.apple.iphone-14-plus-1",
                        "modelName" : "iPhone 14 Plus",
                        "name" : "iPhone 14 Plus",
                        "ignored" : false
                      },
                      {
                        "ignored" : false,
                        "modelCode" : "MacBookPro17,1",
                        "simulator" : false,
                        "modelName" : "MacBook Pro",
                        "operatingSystemVersion" : "13.4.1 (22F82)",
                        "identifier" : "00008103-001654501131001E",
                        "platform" : "com.apple.platform.macosx",
                        "architecture" : "arm64e",
                        "interface" : "usb",
                        "available" : true,
                        "name" : "My Mac",
                        "modelUTI" : "com.apple.macbookpro-13-retina-touchid-late-2020"
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "16.4 (20E247)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPhone14,7",
                        "identifier" : "33217303-A605-4F2C-84BE-8FB162C8EF44",
                        "architecture" : "arm64",
                        "modelUTI" : "com.apple.iphone-14-1",
                        "modelName" : "iPhone 14",
                        "name" : "iPhone 14",
                        "ignored" : false
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "16.4 (20E247)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPad13,18",
                        "identifier" : "02E6B782-51B4-4813-94AA-D2090326B476",
                        "architecture" : "arm64",
                        "modelUTI" : "com.apple.ipad-10-1",
                        "modelName" : "iPad (10th generation)",
                        "name" : "iPad (10th generation)",
                        "ignored" : false
                      }
                    ]

                    2023-07-30 16:41:39.098 xcdevice[24382:182912] DVTCoreDeviceEnabledState:
                    DVTCoreDeviceEnabledState_Disabled set via user default
                    (DVTEnableCoreDevice=disabled)

```

And I've run a `flutter doctor` in case that helps...
```
flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.10.6, on macOS 13.4.1 22F770820d darwin-arm64 (Rosetta), locale en)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.3.1)
[✓] Android Studio (version 2022.2)
[✓] VS Code (version 1.80.2)
[✓] Connected device (2 available)
    ! Error: David’s Apple Watch has recently restarted. To use David’s Apple Watch with Xcode, unlock it and its companion iPhone (code 19)
[✓] Network resources

• No issues found!
```

**Comment by PaulTR** [comment]
So I wasn't able to replicate this. My steps:

`flutter create my_app`
`flutter pub add tflite_flutter`
Edit ios/Runner.xcworkspace to attach my dev signing certificate
Clicking run from xcode with my iPad hooked up

Are you able to walk us through the steps you've taken?

**Comment by davidd647** [comment]
Funny... I hadn't tried running the app directly from Xcode... That works!

I still can't run it from VS Code or the terminal (still hangs on "Running pod install...", but at least I've got something working now.

Thanks!

**Comment by davidd647** [comment]
Solved... 🤦‍♂️

Silly me! Nothing was hanging. Tensorflow is just a big pod to download.

I tried a `pod install --verbose`, and just left without worrying about keeping my phone from going into sleep mode. Took about 10 minutes to finish. Here's more about what happened in case anyone is curious:


After the `pod install --verbose`, a few pods installed as per usual, but TensorFlowLiteSwift seemed to just sit on the screen - nothing happening. I was reading through what was happening.
```pod install --verbose

...

-> Installing TensorFlowLiteSwift (2.12.0)
 > Git download
 > Git download
     $ /usr/bin/git clone https://github.com/tensorflow/tensorflow.git
     /var/folders/gj/wg7f08sd3c5ck_y8hf80zyx00000gn/T/d20230803-47318-at45q8 --template=
     Cloning into '/var/folders/gj/wg7f08sd3c5ck_y8hf80zyx00000gn/T/d20230803-47318-at45q8'...```

I noticed that Cocoapods seemed to be downloading the entire tensorflow github repo, which I thought was strange, and it's possible that the pod is just much bigger than I imagined.
